canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.scss

```diff
@@ -22,7 +22,7 @@
 	width: 0;
 	height: 100%;
 	background-color: $ui-01;
-	overflow: hidden;
```

Comment:
With these changes I can no longer scroll a category with a mouse.  Why did these need to be changed?

Reply:
I updated value of `overflow` to fix this issue - 
Long tooltips are cut off. I will try to find another way to fix this. 
![Screen Shot 2020-10-30 at 9 50 55 AM](https://user-images.githubusercontent.com/25124000/97733814-8e9c7680-1a95-11eb-890c-2d5e25a80e5c.png)